src/Values/Timeout.php

```diff
@@ -21,6 +32,22 @@ public function toArray(): array
         ];
     }
 
+    /**
+     * @param Timeout|null $other
+     * @return static
+     */
+    public function add(?Timeout $other = null): static
```

Comment:
For final classes `self` is the same as `static`. Just FYI, no need to change it.